src/test/java/com/amazon/ion/impl/bin/IonManagedWriter_1_1_Test.kt

```diff
@@ -245,6 +245,54 @@ internal class IonManagedWriter_1_1_Test {
         assertEquals(expected, actual)
     }
 
+    @Test
+    fun `when a system macro is shadowed, it should be written using the system e-exp syntax`() {
+        val expected = """
+            $ion_1_1
+            $ion_encoding::((macro_table (macro make_string () "make")))
+            (:make_string)
+            (:$${"ion::make_string"} (: "a" b))
```

Comment:
```suggestion
            (:$ion::make_string (: "a" b))
```

Why `${"ion::make_string"}`?
At first I  thought this just needed addition to the " annoying to use with Kotlin's string substitution" set at the beginning of the file, but then I saw that `ion` was already there. I just checked and the test passes with this suggestion applied.